src/components/study/Registration.vue

```diff
@@ -25,27 +29,29 @@
 import MainNav from './MainNav.vue'
 import store from '../store'
 import axios from 'axios';
+import { VueTelInput } from 'vue-tel-input'
 
 export default {
   name: 'StudyRegistration',
-  components: { MainNav },
+  components: { MainNav, VueTelInput },
   data() {
     return {
-      phone: '',
-      showAndroid: window.queryParams.android === "true"
+      phoneValue: '',
+      countryCodes: ['CA','US','NL'],
+      defaultCountry: this.$t('registration-screen.default-region-code'),
```

Comment:
Make Canada, US, and Netherlands the only available options.  Set the default region code based on the locale (in nl it's set to NL, in en it's US).